microsoft-365/security/mtp/top-scoring-industry-tests.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Top scoring in industry tests (AV-TEST, AV Comparatives, SE Labs, MITRE ATT&CK)
+title: Microsoft Threat Protection - Top scoring in industry tests
```

Comment:
Shouldn't it be the title, and then the product? "Top scoring in industry tests - Microsoft Threat Protection"